src/components/MatchPredictionsBar.vue

```diff
@@ -84,6 +87,11 @@ export default {
   },
 
   computed: {
+    rightSideBorderIfStarted() {
+      if (this.match.status === 'started') {
+        return 'border-r border-gray-300'
+      } else return ''
+    },
```

Comment:
What is this border doing? Tried finding it but it looks the same as the other percentage games

Reply:
@dmbf29 this border is separating the two inner bars (for home and away). It is needed as there isn't a draw bar which usually contains the border, and one of the bars is not green which usually provides a natural separation.

Reply:
This was just a quick fix that I added but I think a better solution is needed as it could lead to double borders during a group stage game that is 'started' as there would be this in addition to the border on the draw portion of the bar.